app/models/dataservice/process_external_activity_data_job.rb

```diff
@@ -47,10 +52,21 @@ def perform
         else
           raise UnknownRespose.new("type: #{student_response["type"]}\nContent: #{content}")
         end
+
+      #
       # We could broaden this to StdErr, but this should catch most cases
       # which will be NPE-like cases
-      rescue NameError => stderr
-        log_exception(stderr, learner_id, student_response)
+      #
+      rescue NameError => e
+
+        # Delayed::Worker.logger.debug("*** rescue #{e}")
+        # Delayed::Worker.logger.debug("*** learner_id #{learner_id}")
+        # Delayed::Worker.logger.debug("*** student_ response " <<
+        #                               "#{student_response}")
+        # Delayed::Worker.logger.error e.backtrace.join("\n")
+
+        log_exception(e, learner_id, student_response)
+
```

Comment:
It could be useful to uncomment these lines is they aren't going to break something. Then we would get these types of errors reported both in new relic and hopefully in the production logs as well. Now that we've fixed a couple of issues, hopefully these types of errors will become more meaningful.